AirCasting/SessionViews/MultiSliderView.swift

```diff
@@ -6,19 +6,24 @@
 //
 
 import SwiftUI
+import Resolver
 
 struct ThresholdsSliderView: View {
     
     @ObservedObject var threshold: SensorThreshold
+    @Binding var selectedStream: MeasurementStreamEntity?
     
     var body: some View {
-        MultiSliderView(thresholds: threshold.rawThresholdsBinding)
+        if let selectedStream = selectedStream {
+            MultiSliderView(thresholds: threshold.rawThresholdsBinding, selectedStream: selectedStream)
+        }
     }
 }
 
 struct MultiSliderView: View {
-    
+    @InjectedObject private var userSettings: UserSettings
     @Binding var thresholds: [Float]
+    @ObservedObject var selectedStream: MeasurementStreamEntity
```

Comment:
I think you can remove the `@ObservedObject` wrapper. I checked, and everything seems to be working fine without it.